gather-external-docs fails due to moved OH2 addons feature file

In https://github.com/openhab/openhab2-addons/pull/5018 the openhab2-addons feature file was moved:

`features/openhab-addons/src/main/feature/feature.xml`
→ 
`features/karaf/openhab-addons/src/main/feature/feature.xml`

---

As a result the [gather-external-docs](https://ci.openhab.org/view/Documentation/job/gather-external-docs/194/console) Jenkins job now fails with:

`[ERROR] Failed to execute goal org.codehaus.groovy.maven:gmaven-plugin:1.0:execute (process-addon-files) on project docs: java.io.FileNotFoundException: /var/jenkins_home/workspace/gather-external-docs/.external-resources/openhab2-addons/features/openhab-addons/src/main/feature/feature.xml (No such file or directory) -> [Help 1]`


This also causes https://github.com/openhab/openhab-docs/issues/908.

Can you have a look at this @openhab/documentation-maintainers ?

--- a/process_addons.groovy
+++ b/process_addons.groovy
@@ -9,7 +9,7 @@ def collect_feature_xml = { features, xml, attrs ->
 
 def collect_features = { features ->
     collect_feature_xml(features, '.external-resources/openhab-distro/features/addons/src/main/feature/feature.xml',                  ['install': 'auto',   'since': '2x'])
-    collect_feature_xml(features, '.external-resources/openhab2-addons/features/openhab-addons/src/main/feature/feature.xml',         ['install': 'auto',   'since': '2x'])
+    collect_feature_xml(features, '.external-resources/openhab2-addons/features/karaf/openhab-addons/src/main/feature/feature.xml',         ['install': 'auto',   'since': '2x'])
     collect_feature_xml(features, '.external-resources/openhab1-addons/features/openhab-addons/src/main/feature/feature.xml',         ['install': 'auto',   'since': '1x'])
     collect_feature_xml(features, '.external-resources/openhab1-addons/features/openhab-addons-legacy/src/main/feature/feature.xml',  ['install': 'legacy', 'since': '1x'])
 }

